README.adoc

```diff
@@ -72,12 +205,15 @@ Logging output is displayed. The service should be up and running within a few s
 
 == Test the application
 
-Now that the application is running, you can test it.
+Now that the application is running, you can test it. Open http://localhost:8080, type a message
```

Comment:
The instructions don't talk about creating the `index.html` page. So, if you just followed the instructions, this won't exactly work.